src/setup.sh

```diff
@@ -294,19 +336,32 @@ function check_prerequisites {
 # Installs all the dependencies for docker-osx-dev.
 #
 function install_dependencies {
-  log_info "Updating HomeBrew"
-  brew update
+  if is_brew_installed ; then
+
+    log_info "Updating HomeBrew"
+    brew update
+
+    brew_install "caskroom/cask/brew-cask" "Cask" "" false
+    brew_install "virtualbox" "VirtualBox" "VBoxManage" true
+    brew_install "boot2docker" "Boot2Docker" "boot2docker" false
+    brew_install "docker-compose" "Docker Compose" "docker-compose" false
+    brew_install "fswatch" "fswatch" "fswatch" false
+    brew_install "coreutils" "GNU core utilities" "greadlink" false
 
-  brew_install "caskroom/cask/brew-cask" "Cask" "" false
-  brew_install "virtualbox" "VirtualBox" "VBoxManage" true
-  brew_install "boot2docker" "Boot2Docker" "boot2docker" false
-  brew_install "docker-compose" "Docker Compose" "docker-compose" false
-  brew_install "fswatch" "fswatch" "fswatch" false
-  brew_install "coreutils" "GNU core utilities" "greadlink" false
+  elif is_port_installed ; then
+
+    log_info "Updating MacPorts"
+    sudo port selfupdate
+
+    port_install "virtualbox" "VirtualBox" "VBoxManage"
```

Comment:
Good call, but you have to consider possibly different naming conventions
as well.

>